File: ftc_app-master/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/tests/TurnTest.java
package org.firstinspires.ftc.teamcode.opmodes.tests;

import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;

import org.firstinspires.ftc.teamcode.robots.TestRobot;

@Autonomous(name = "TB - Turn Test", group = "Tests")
public class TurnTest extends LinearOpMode {

    @Override public void runOpMode() throws InterruptedException {
        TestRobot robot = new TestRobot();
        robot.setup(this);
        waitForStart();
        robot.turn(0.45, 90);
    }

}

File: ftc_app-3.5/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/tests/TurnTest.kt
package org.firstinspires.ftc.teamcode.opmodes.tests

import com.qualcomm.robotcore.eventloop.opmode.Autonomous
import com.qualcomm.robotcore.eventloop.opmode.Disabled
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode
import org.firstinspires.ftc.teamcode.robots.RelicRecoveryRobot

@Disabled
@Autonomous(name = "Turn Test", group = "Tests")
class TurnTest : LinearOpMode() {

    @Throws(InterruptedException::class)
    override fun runOpMode() {
        val robot = RelicRecoveryRobot()
        robot.setup(this)
        waitForStart()
        robot.turn(0.45, 90)
        robot.turn(0.45, -90)
    }

}
